src/toolbar/toolbar.jsx

```diff
@@ -46,21 +67,36 @@ const Toolbar = React.createClass({
     return this.state.muiTheme.toolbar;
   },
 
+  getSpacing() {
```

Comment:
I suppose that the `getSpacing`  and the `getTheme` are only called in the `getStyles`.
In this case, shouldn't we remove those two methods and add two variables in the `getStyles`?